src/coredata.cs

```diff
@@ -525,7 +571,7 @@ interface NSIncrementalStore {
 
 		[Static]
 		[Export ("identifierForNewStoreAtURL:")]
-		NSObject IdentifierForNewStoreAtURL (NSUrl storeURL);
+		NSObject IdentifierForNewStoreAtUrl (NSUrl storeUrl);
```

Comment:
breaking change, not worth fixing (you can put the `Url` version in `XAMCORE_4_0`)